assets/js/blocks/shared/styles/style.scss

```diff
@@ -0,0 +1,15 @@
+.wc-block-attribute-filter.style-dropdown,
+.wc-block-rating-filter.style-dropdown,
+.wc-block-stock-filter.style-dropdown {
+	.single-selection {
+		.components-form-token-field__token + input[type="text"].components-form-token-field__input {
+			opacity: 0;
+		}
+
+		.components-form-token-field__input-container.is-active {
+			.components-form-token-field__token + input[type="text"].components-form-token-field__input {
+				opacity: 1;
+			}
+		}
+	}
+}
```

Comment:
Not needed for this PR, but I wonder if at some point we should create a `filters` directory under `assets/js/blocks/`. That would make it easier to share styles and logic between them, like this one. Similar to what was done with Featured Product and Featured Category, which have a common folder (`featured-items`).

Reply:
I created a `assets/js/blocks/shared/styles/style.css` file (and added a mixin dedicated to filters), but in the near future we should make a dedicated path (`assets/js/blocks/filters/*`) with shared JS and SCSS code. There's too much code overlap between three filters to not combine it 😆  The issue is already created: https://github.com/woocommerce/woocommerce-blocks/issues/7889 and that's definitely the direction we should follow.